MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -258,6 +260,208 @@ The [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.
+                      XBindingPath="XValues"
+                      YBindingPath="YValues"/>
+
+    <chart:LineSeries ItemsSource="{Binding Data2}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="XValues"
+                      YBindingPath="YValues"/>
+
+    <chart:LineSeries ItemsSource="{Binding Data3}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="XValues"
+                      YBindingPath="YValues"/>
+                      
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+ChartTrackballBehavior trackball = new ChartTrackballBehavior();
+chart.TrackballBehavior= trackball;
+. . .
+LineSeries series1 = new LineSeries();
+series1.ItemsSource = new ViewModel().Data1;
+series1.XBindingPath = "XValues";
+series1.YBindingPath = "YValues";
+
+LineSeries series2 = new LineSeries();
+series2.ItemsSource = new ViewModel().Data2;
+series2.XBindingPath = "XValues";
+series2.YBindingPath = "YValues";
+
+LineSeries series3 = new LineSeries();
+series3.ItemsSource = new ViewModel().Data3;
+series3.XBindingPath = "XValues";
+series3.YBindingPath = "YValues";
+
+DataTemplate trackballLabelTemplate = new DataTemplate(() =>
+{
+    HorizontalStackLayout horizontalStackLayout = new HorizontalStackLayout
+    {
+        Spacing = 5
+    };
+
+    Image image = new Image
+    {
+        Source = "image.png",
+        WidthRequest = 20,
+        HeightRequest = 20
+    };
+
+    Label label1 = new Label
+    {
+        FontSize = 15,
+        HorizontalOptions = LayoutOptions.Center,
+        TextColor = Colors.White
+    };
+    label1.SetBinding(Label.TextProperty,"DataItem.Name");
+
+    Label label2 = new Label
+    {
+        Text = " : ",
+        FontSize = 15,
+        HorizontalOptions = LayoutOptions.Center,
+        TextColor = Colors.White
+    };
+
+    Label label3 = new Label
+    {
+        FontSize = 15,
+        HorizontalOptions = LayoutOptions.Center,
+        TextColor = Colors.White
+    };
+    label3.SetBinding(Label.TextProperty," DataItem.YValues", stringFormat: "{0}M");
+
+    horizontalStackLayout.Add(image);
+    horizontalStackLayout.Add(label1);
+    horizontalStackLayout.Add(label2);
+    horizontalStackLayout.Add(label3);
+
+    return horizontalStackLayout;
+});
+
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series2.TrackballLabelTemplate = trackballLabelTemplate;
+series3.TrackballLabelTemplate = trackballLabelTemplate;
+
+chart.Series.Add(series1);
+chart.Series.Add(series2);
+chart.Series.Add(series3);
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Trackball Label Template support in MAUI chart](Trackball_images/maui_chart_trackball_label_template.png)
+
+## Customize the Axis Label with DataTemplate
+
```

Comment:
Trackball axis label template - change title.

Reply:
Changed the title